### DISCUSSION TITLE
关于打印窗口大小变化的问题

### DISCUSSION THREAD

**Discussion by lhyyy1** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#54] [Q&A]
是要利用test/concurrency_visualizer.py来打印吗？文档中似乎没有说concurrency_visualizer.py怎么用

**Comment by TANSixu** [comment]
Hi，窗口大小需要自己在peer的实现中记录并绘图，不是用concurrency_visualizer。concurrency_visualizer是用于concurrency的可视化，可参考 #33 。由于在网络中无法直接获得同学们实现准确的窗口大小，所以需要自己画一下哈。

**Comment by LoserXj** [comment]
请问自己画的话，横坐标有要求吗，每次cwnd对应的横坐标可以是1，2，3，4......吗？还是得对应时间呢

**Reply by TANSixu** [reply]
hi，没有要求，只要能展示出变化趋势即可。
